### PR TITLE
ITS Checks: info message for shifters from .json

### PR DESCRIPTION
**Pull request by IsakovAD**
Added possibility to display info message from the .json on top of MOs

### CODE REVIEW

File: Modules/ITS/itsFee.json [hunk 1]
```diff
@@ -51,11 +51,13 @@
                 "detectorName": "ITS",
                 "checkParameters": {
                   "skipbinstrg": "",
-                  "skipfeeids": "",
+                  "skipfeeids": "12,11,10",
```
Comment (iravasen) [inline Modules/ITS/itsFee.json:54]: remove 12,11,10. I guess comes from a test

File: Modules/ITS/itsFhr.json [hunk 2]
```diff
@@ -75,7 +75,10 @@
                 "checkParameters": {
                     "fhrcutIB": "0.01",
                     "fhrcutOB": "0.0001",
-                    "skipbins": ""
+                    "skipbins": "",
+                    "plotWithTextMessage": "General/General_Occupancy,General/ErrorPlots",
+                    "textMessage": "test message occ,test message errors"
```
Comment (iravasen) [inline Modules/ITS/itsFhr.json:80]: also these two come from a test I guess.

**Comment by iravasen** [comment]
@IsakovAD great job! Thanks a lot. Can you apply the same modification also to the DecodingErrorCheck? Please note that in this PR you are including also the modifications for the QCchecks on ITS sectors which I now merged. I see we got conflicts. Can you please resolve them? Sorry for that, I note it only now.

File: Modules/ITS/itsFee.json (hunk 1 above)
Comment (IsakovAD) [reply Modules/ITS/itsFee.json]: done

File: Modules/ITS/itsFhr.json (hunk 2 above)
Comment (IsakovAD) [reply Modules/ITS/itsFhr.json]: done

**Comment by IsakovAD** [comment]
@iravasen all comments were implemented

**Comment by Barthelemy** [comment]
@iravasen I let you merge

**Comment by Barthelemy** [comment]
@iravasen we release later today (short week). Sorry for the late warning.

**Review by iravasen — APPROVED** [review]

**Comment by IsakovAD** [comment]
@Barthelemy @knopers8 

Could you please add your review?

**Pull request merged**